### PR TITLE
Ref schema update

### PR DESCRIPTION
**Pull request by Shakir-Sopra**
Hi Warren 

As advised updated the environment specific file and removed the code of creation of schema and relocation of tables code to /Dev branch and its again ready for your review.

Thanks
Shakir

### CODE REVIEW

File: DEV/dev.db.sql [hunk 1]
```diff
@@ -1 +1,13 @@
...
+--Creating cqcref schema and removing the unused schema's.
+CREATE SCHEMA cqcref
+    AUTHORIZATION sfcadmin;
+
+alter table cqc.pcodedata set schema cqcref;
+alter table cqc.location set schema cqcref;
+
+
+DROP SCHEMA cqctst cascade;
+DROP SCHEMA cqctsttst cascade;
```
Comment (wozzer72) [inline DEV/dev.db.sql]: Whatbout `sfcfuldata` schema?

**Comment by wozzer72** [comment]
Where are the tst and uat files?

**Comment by Shakir-Sopra** [comment]
I will update the same in test and UAT folder and let you know once done.

Thanks
Shakir

**Comment by Shakir-Sopra** [comment]
@wozzer72 

I have updated TST and UAT files as well.

Thanks
Shakir

**Comment by Shakir-Sopra** [comment]
@wozzer72 

I have put a drop statement for sfcfuldata for test environment and its ready for your review.

I completely agree with you, reference data from AddressPlus shouldn't be replicated in every environment.

Thanks
Shakir

**Comment by Shakir-Sopra** [comment]
@wozzer72 
Thanks for approving changes now merging the branch.

**Pull request merged**